xml/depl_maintenance.xml

```diff
@@ -559,7 +559,7 @@
      <informalfigure>
       <mediaobject>
        <imageobject role="fo">
-        <imagedata fileref="depl_upgrade_initial.png" width="100%" format="png"/>
+        <imagedata fileref="depl_upgrade_initial.png" width="75%" format="png"/>
```

Comment:
The width for fo should be 100% for all screenshots here (or, 99% if that does not work).

Sorry to be a pain, but the screenshots are a liitle too wide and on most of them there is a lot of unused gray space. Can you try to shoot them with a height that fits the content and less width (by choosing the minimal browser width that is possible?). The bigger the screenshots are, the harder it is to read them in PDF...